bot/plotting/random_plot_generator.py

```diff
@@ -98,10 +134,41 @@ def random_plot_generator(
 
             elif choice == 1:
                 (
-                    cycleReceived,
+                    cycle_received,
                     number,
                 ) = experiment_generator()
-                experiment = pybamm.Experiment(cycleReceived * number)
+                if testing:
+                    number = 10
+                if number > 3 and plot_summary_variables:
+
+                    experiment = pybamm.Experiment(
+                        cycle_received * number, termination="80% capacity"
+                    )
+                    (
+                        sim,
+                        solution,
+                        parameter_values
+                    ) = experiment_solver(
+                        model=model,
+                        experiment=experiment,
+                        chemistry=chemistry,
+                        solver=solver
+                    )
+                    generate_summary_variables(solution)
+                    return (
+                        model,
+                        parameter_values,
+                        None,
+                        chemistry,
+                        solver,
+                        True,
+                        cycle_received,
+                        number,
+                        False,
+                    )
```

Comment:
it might become hard to keep track of the order of these long return lists. Fine for now but something to bear in mind (maybe dictionary better)

Reply:
These variables are dummy ones, for now, they contain all the information that I initially thought would be tweeted with the plot. I will keep the dictionary option in mind for the final variables.